### PR TITLE
Add user management docs

### PR DESCRIPTION
**Pull request by gomathyk**

### CODE REVIEW

File: en/docs/concepts/user-management/user-management.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Introduction to User Management 
+
+In a typical organization, there are several employees who are responsible for accomplishing a versatile set of tasks on a daily basis with the help of different applications. Most of the times, even the same application is used in different ways by employees who have differenet job roles. Managing who accesses which application for what purposes can be an overhead for the administration. This is why user management is important.
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/user-management.md:3]: times--> time

differenet --> different

File: en/docs/concepts/user-management/users.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+# Users
+
+A user is the digital representation of a physical user who interacts with applications. A user can be a digital identity of a human or a machine. A user can be recognized by a number of attributes. These are called **user claims**. User claims are data that defines the user, i.e., anything the user is, owned by, or associated with.
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/users.md:3]: I think we have renamed "claims" to "attributes" in the new portals. If so, let's not use the term "claims" in the concept docs either

File: en/docs/concepts/user-management/users.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+A unique **User account** is created for each user. Then they are assigned roles with a certain set of permissions. We will further discuss roles and permissions in the [next section](../../concepts/user-management/roles-and-permissions.md).
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/users.md:5]: Let's try and avoid using "you" and "we" in official docs wherever we can. In this case, we can rephrase it to something like "Roles and permissions are explained further in the next section"

File: en/docs/concepts/user-management/users.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+---
+
+## Add users
+
+You can create or add users to a user management console in more than one way. Following are a few way users can be created or added. 
+
+- Using the user management console - Here, the user is created by the admin using the user management application. 
+
+- Using the **ask password** option - This option is widely preferred because the admin does not have to remember passwords of created users. This can be done by prompting the user to create a password for themselves while being on-boarded or administrators on-boarding user accounts to the system along with a default password, sending an email link to the user with the created account and asking the user to confirm the correctness of the account created by the admin.
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/users.md:17]: lets break this sentence if possible to make it clearer?

File: en/docs/concepts/user-management/users.md [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+## Update users
+
+User details might need to be updated from time to time to maintain the precision of data in the user management system. For example, when the user communicates a change in any of the user profile details like the user's email address or phone number, it can be updated by the admin using the user management console, a SCIM request with the new values, or a SOAP request. It can also be updated by users themselves if there is a user portal application provided by the user management system. 
+
+---
+
+## Import users 
+
+In addition to manually adding individual users, you can import multiple users in bulk. This is usually done by first exporting all the relevant user details to a comma-separated values (.csv) file or Microsoft Excel (.xls) file, and then importing this file with all the information to the user management system. 
+
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/users.md:36]: shall we link to the relevant guides from these sections?

File: en/docs/concepts/user-management/users.md [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+---
+
+## Update users
+
+User details might need to be updated from time to time to maintain the precision of data in the user management system. For example, when the user communicates a change in any of the user profile details like the user's email address or phone number, it can be updated by the admin using the user management console, a SCIM request with the new values, or a SOAP request. It can also be updated by users themselves if there is a user portal application provided by the user management system. 
+
+---
+
+## Import users
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/users.md:33]: IMO import users should come as a point under the previous section where we list ways of creating/adding users

File: en/docs/concepts/user-management/roles-and-permissions.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+# Roles and Permissions
+
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/roles-and-permissions.md:2]: shall we start the doc by simple definitions of what is a role and what is a permission? WDYT?

File: en/docs/concepts/user-management/user-management.md (hunk 1 above)
Comment (sherenewso2) [reply en/docs/concepts/user-management/user-management.md:3]: I think we can add a little bit more about why user management is important and make it a separate sub topic. Let's start this page with a simple definition of what we mean by user management? 

after explaining the concept, let's dive into how user management works with WSO2 IS specifically. 

(check out some competitor docs for ideas on structure : https://www.okta.com/blog/2019/01/user-management/)

File: en/docs/guides/identity-lifecycles/update-profile.md [hunk 8]
```diff
@@ -101,7 +101,7 @@ admin services, see [Calling Admin Services](insert-calling admin services).
 ----
 
 !!! info "Related Topics"
-    - [Concept: User Profiles](TODO:insert-link-to-concept)
```
Comment (sherenewso2) [inline en/docs/guides/identity-lifecycles/update-profile.md:104]: let's add something about what a user profile is also somewhere in the concept doc?

File: en/docs/concepts/user-management/users.md (hunk 5 above)
Comment (gomathyk) [reply en/docs/concepts/user-management/users.md:36]: Since it is an overview of all the functionalities and the page is also relatively short, I have mentioned them in the "related topics" section in the end. I found it messy to link it twice within a short page. WDYT?

File: en/docs/guides/identity-lifecycles/update-profile.md (hunk 8 above)
Comment (gomathyk) [reply en/docs/guides/identity-lifecycles/update-profile.md:104]: Details regarding the user account are added to the user management doc. Not sure if we need to add user profile information separately. Let's discuss this separately?

File: en/docs/concepts/user-management/roles-and-permissions.md [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+Roles are groups of users categorized based on the tasks they are expected to perform. In a typical organization, roles can be based on (but not limited to) the department a user works for or the user's designation. Permissions define a targetted set of accesses a user is provided with to perform tasks that are a part of their role.
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/roles-and-permissions.md:3]: targetted --> targeted

File: en/docs/concepts/user-management/roles-and-permissions.md [hunk 10]
```diff
@@ -0,0 +1,34 @@
+# Roles and Permissions
+
+Roles are groups of users categorized based on the tasks they are expected to perform. In a typical organization, roles can be based on (but not limited to) the department a user works for or the user's designation. Permissions define a targetted set of accesses a user is provided with to perform tasks that are a part of their role. 
+
+Each of these roles have a defined set of permissions associated with them. Users who are assigned to a particular role are authorized to perform activities that align with the permissions of that particular role. Users can be assigned one or many roles. 
+
+WSO2 Identity Server enables role-based access. Simply put, access is granted based on a user's roles rather than each user's individual attributes. This makes it simpler to assign or de-assign permissions.
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/roles-and-permissions.md:7]: de-assign --> unassign ?

File: en/docs/concepts/user-management/user-management.md [hunk 11]
```diff
@@ -0,0 +1,31 @@
+# Introduction to User Management 
+
+User management, in simple terms, is a process of efficiently managing users within a user store. User management is vital for any application since a user who is not authenticated and/or authorized properly will not be able to access the applicationn and use it in a way that serves the user's requirements. On the other hand, users might be bombarded with unnecessary features of the application and information if the application cannot provide them with a filtered version as per their need.
```
Comment (sherenewso2) [inline en/docs/concepts/user-management/user-management.md:3]: applicationn --> application

**Review by sherenewso2 — APPROVED** [review]
